samples/javascript/E4_SendSmsChallenge/index.js

```diff
@@ -1,17 +1,19 @@
 const seedrandom = require("seedrandom");
 const uuidv1 = require("uuid/v1");
 
-module.exports = function (context, phoneNumber) {
+module.exports = async function (context, phoneNumber) {
     // Get a random number generator with a random seed (not time-based)
     const rand = seedrandom(uuidv1());
```

Comment:
Could `const rand = seedrandom(uuidv1());` be pulled outside of the function scope? not sure if that generates bad behavior, but if possible may be (slightly, this is a nit) better

Reply:
This should be fine. Thanks for catching that.